### DISCUSSION TITLE
LTC Not Working

### DISCUSSION THREAD

**Discussion by chaudhuri-ab** [gopro/labs#449] [Q&A]
Do I need to enable a setting to jam sync via 3.5mm line in on the Go Pro. I have signal from a Zoom M4 going into the line in on a Hero 9 Black, but am not getting the magic time codes popping up. I verified that the LTC is active from the M4 just by plugging in head phones, but other than that was thinking it would just be plug and play.

Is there a setting that I need to enable via QR Control before the Go Pro response to LTC coming into the Line In?

**Comment by dnewman-gpsw** [comment]
You might need to have voice activation running on some camera models.  Nothing else needs to be done, but I believe only one channel works (left or right, sorry I don't remember.)
